active/ListViewGridView/listviewgridview-designupdates.md

```diff
@@ -0,0 +1,495 @@
+
+<StaticResource x:Key="GridViewItemSelectedDisabledBorderBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxDisabledBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxSelectedBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxSelectedPointerOverBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxSelectedPressedBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxSelectedDisabledBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxBorderBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxPointerOverBorderBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxPressedBorderBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemCheckBoxDisabledBorderBrush"  ResourceKey="SystemAccentColor"/>
+
+<StaticResource x:Key="GridViewItemSelectedInnerBorderBrush"  ResourceKey="SystemAccentColor"/>
+
+<CornerRadius x:Key="GridViewItemCheckBoxCornerRadius">3</CornerRadius>
+
+<CornerRadius x:Key="GridViewItemCornerRadius">4</CornerRadius>
+
+```
+
+# API Details
+
+```cs
+
+unsealed runtimeclass ListViewItemPresenter
+    : Controls.ContentPresenter
+{
+    // ... 
+    // existing 
+    // ...
+
+    [contract(Windows.Foundation.UniversalApiContract, 13)]
+    {
+        Brush PointerOverBackground;
+        Brush SelectedBackground;
+        Brush SelectedPointerOverBackground;
+        Brush SelectedPressedBackground;
+        Brush PressedBackground;
+        Brush SelectedDisabledBackground;
+        Brush CheckPressedBrush;
+        Brush CheckDisabledBrush;
+        Brush CheckBoxPointerOverBrush;
+        Brush CheckBoxPressedBrush;
+        Brush CheckBoxDisabledBrush;
+        Brush CheckBoxSelectedBrush;
+        Brush CheckBoxSelectedPointerOverBrush;
+        Brush CheckBoxSelectedPressedBrush;
+        Brush CheckBoxSelectedDisabledBrush;
+        Brush CheckBoxBorderBrush;
+        Brush CheckBoxPointerOverBorderBrush;
+        Brush CheckBoxPressedBorderBrush;
+        Brush CheckBoxDisabledBorderBrush;
+        CornerRadius CheckBoxCornerRadius;
+        CornerRadius SelectionIndicatorCornerRadius;
+        Boolean SelectionIndicatorVisualEnabled;
+        Controls.Primitives.ListViewItemPresenterSelectionIndicatorMode SelectionIndicatorMode;
+        Brush SelectionIndicatorBrush;
+        Brush SelectionIndicatorPointerOverBrush;
+        Brush SelectionIndicatorPressedBrush;
+        Brush SelectionIndicatorDisabledBrush;
+        Brush SelectedBorderBrush;
+        Brush SelectedPressedBorderBrush;
+        Brush SelectedDisabledBorderBrush;
+        Brush SelectedInnerBorderBrush;
+        Brush PointerOverBorderBrush;
+
+        static DependencyProperty SelectedDisabledBackgroundProperty{ get; };
```

Comment:
Is this supposed to have a DependencyProperty for each property? It's missing some.

Reply:
Is there a reason why those styling resources have become a resource? I think a lightweight styling resource makes more sense here.

Reply:
I think in a previous version I had denoted with comments in the API details section which APIs are new and which are changed. Those first five APIs currently exist, but they're being changed with this spec. I did not provide the DependencyProperty for those five as I don't think those are changing.